spec/features/home_page_spec.rb

```diff
@@ -2,13 +2,12 @@
 
 feature "Visiting the homepage", :js do
   scenario "has errors" do
-    visit "/"
+    logger = double(:logger, tagged: nil)
```

Comment:
What are your thoughts on turning this into a `StringIO` and testing the output?